controllers/replication.storage/volumereplication_controller.go

```diff
@@ -363,13 +372,34 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	}
 
 	instance.Status.LastCompletionTime = getCurrentTime()
+
+	var requeueForInfo bool
+	if instance.Spec.ReplicationState == replicationv1alpha1.Primary {
```

Comment:
@Madhu-1 If an image is in the split-brain state, is there a TS that is usable?

Reply:
AFAIK no during split-brain no sync will happen this is not useful.

Reply:
In split-brain, I do not assume one of the images is marked **Primary**? If that is the case, the lastSyncTime should be the time of the last successful sync, maybe.

Reply:
since we are updating it in every interval, I guess if we reach split brain case, it would already have that, cc @ShyamsundarR

Reply:
The question really is what does RBD report, and then what should the proto report. In the case of split-brain we should report not-synced time (which IMHO is the "0" timestamp), or if available the last time it was synced...

Reply:
I agree, on a sync failure (like split-brain), no LastSyncTime should be reported, but ideally an error. Either the automatic rescheduling should stop, or be done with an exponential backoff.

Reply:
When the image is in `replaying` state with RBD we get the additional description, if stopped or split-brain the additional data is not available, and we will hence default to the "0" time stamp. We should be fine with that.